### ISSUE TITLE
catkin, OpenSfM, and LASTools download

### ISSUE THREAD

**Issue by dakotabenjamin** [edgarriba/OpenDroneMap#14]
Th file downloaded for catkin is `indigo-devel.zip`, which will change when the repo updates- wouldn't it be better to download a release?

Edit this applies to more than just catkin, but also OpenSfM and LASTools

**Comment by dakotabenjamin** [comment]
You would have to update the MD5 often- for example now its 6940f006726a2b60512372576ffdab8d (for catkin)

**Comment by edgarriba** [comment]
you are right. Need to fix that, a bit busy this days, PR are accepted :-)

**Comment by dakotabenjamin** [comment]
As per @paulinus comment in #16, how should we pull updates from OpenSfM? Maybe instead of downloading from a release, we should download a zip of a specific commit? That way we are getting a persistent file, but can easily update as needed without asking for a new release tag.
